test/test_wsgi_service.py

```diff
@@ -1,10 +1,10 @@
 """Functional tests for WSGI app for Protocol RPC service API."""
```

Comment:
the wsgi_service module imports from what appears to be a protorpc fork that exists only in the google-cloud-sdk install from what I can tell. The version of protorpc that is installed from standard pypi does not have a wsgi package under it for example. Thus in order to get these tests to run you have to something like

```export PYTHONPATH=/google/google-cloud-sdk/platform/google_appengine/lib/protorpc-1.0/```

But let me know if I'm missing something...